website/docs/r/elb_health.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "opentelekomcloud"
+page_title: "OpentelekomCloud: opentelekomcloud_elb_health"
+sidebar_current: "docs-opentelekomcloud-resource-elb-health"
+description: |-
+  Manages an elastic loadbalancer health resource within OpentelekomCloud.
+---
+
+# opentelekomcloud\_elb\_health
+
+Manages an elastic loadbalancer health resource within OpentelekomCloud.
+
+## Example Usage
+
+```hcl
+resource "opentelekomcloud_elb_loadbalancer" "elb" {
+  name = "elb"
+  type = "External"
+  description = "test elb"
+  vpc_id = "e346dc4a-d9a6-46f4-90df-10153626076e"
+  admin_state_up = 1
+  bandwidth = 5
+}
+
+resource "opentelekomcloud_elb_listener" "listener" {
+  name = "test-elb-listener"
+  description = "great listener"
+  protocol = "TCP"
+  backend_protocol = "TCP"
+  protocol_port = 12345
+  backend_port = 8080
+  lb_algorithm = "roundrobin"
+  loadbalancer_id = "${opentelekomcloud_elb_loadbalancer.elb.id}"
+  timeouts {
+	create = "5m"
+	update = "5m"
+	delete = "5m"
+  }
+}
+
+resource "opentelekomcloud_elb_health" "healthcheck" {
+  listener_id = "${opentelekomcloud_elb_listener.listener.id}"
+  healthcheck_protocol = "TCP"
+  healthcheck_connect_porta = 22
+  healthy_threshold = 5
+  healthcheck_timeout = 25
+  healthcheck_interval = 3
+  timeouts {
+    create = "5m"
+    update = "5m"
+    delete = "5m"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional) The region in which to create the elb health. If
+    omitted, the `region` argument of the provider is used. Changing this
+    creates a new elb health.
+
+* `listener_id` - (Required) Specifies the ID of the listener to which the health
+    check task belongs.
```

Comment:
better to add a "Changing this creates a new elb health" like above to let users know this argument doesn't support updating.